src/components/pages/__tests__/AFPStep.test.tsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import AFPStep from '../AFPStep'
+import { FormContext } from '@/contexts/context'
+import { initialFormState } from '../../FormPage'
+import useErrorHandling from '../../../helpers/useErrorHandling'
+
+// Mock the useErrorHandling hook
+jest.mock('../../../helpers/useErrorHandling', () => ({
+  __esModule: true,
+  default: jest.fn(),
+}))
+
+// Mock the FormButtons component
+jest.mock('../../FormButtons', () => ({
+  __esModule: true,
+  default: jest.fn(() => <div>Mocked FormButtons</div>),
+}))
+
+describe('AFPStep Component', () => {
+  const mockGoToNext = jest.fn()
+  const mockSetState = jest.fn()
+
+  const defaultFormPageProps = {
+    curStep: 1,
+    length: 5,
+    goBack: jest.fn(),
+    onStepChange: jest.fn(),
+    handleSubmit: jest.fn(),
+    goToNext: mockGoToNext,
+  }
+
+  const context = {
+    setState: mockSetState,
+    states: initialFormState,
+    formPageProps: defaultFormPageProps,
+  }
+
+  const mockValidateFields = jest.fn()
+  const mockClearError = jest.fn()
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    ;(useErrorHandling as jest.Mock).mockReturnValue([
+      {},
+      { validateFields: mockValidateFields, clearError: mockClearError },
+    ])
+  })
+
+  const renderComponent = () => {
+    return render(
+      <FormContext.Provider value={context}>
+        <AFPStep />
+      </FormContext.Provider>
+    )
+  }
+
+  test('Burde rendre', () => {
```

Comment:
ref tidligere kommentar, jobbe med descriptions, konsekvent skrift og nøsting av tester i beskrivelser.

Reply:
Test med description "Burde rendre". Klarer vi en bedre beskrivelse enn det, og med samme skrift som de andre?
Hvis hensikten er å teste tekstene og labelene for de ulike feltene kan man sjekke dem mer systematisk og med en passende beskrivelse.